modules/analyse/src/main/Advice.scala

```diff
@@ -110,7 +110,6 @@ private[analyse] object MateAdvice {
         case MateLost if nextCp > 999 => Inaccuracy
         case MateLost if nextCp > 700 => Mistake
         case MateLost => Blunder
-        case MateDelayed => Inaccuracy
```

Comment:
You have to define the behaviour of `MateDelayed`, or else this will break at runtime.